tlslite/tlsrecordlayer.py

```diff
@@ -147,6 +147,9 @@ def __init__(self, sock):
         #Limit the size of outgoing records to following size
         self.recordSize = 16384 # 2**14
 
+        # Indicator for heartbeat extension
+        self.heartbeat = False
```

Comment:
Variables with names `heartbeat` for our mode and `heartbeatOppositeSide` for other side is ok? Is this better, than `heartbeatClientMode` and `heartbeatServerMode`, in which we dont know, which mode is our, until we find if we are client/server?